docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,173 @@ description: Next.js supports built-in image optimization, as well as third part
+
+```js
 import Image from 'next/image'
+import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
         alt="Picture of the author"
-        width={500}
-        height={500}
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // placeholder="blur" // Optional blur-up while loading
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
-
-export default Home
 ```
 
-## Image Imports
-
-You can statically `import` images that live in your project. Dynamic `await import()` or `require()` are _not_ supported.
+### Remote Images
 
-With static `import`s, you only need to provide the `src` prop. The `width`, `height`, and `blurDataURL` props will automatically be populated. Alt text is still needed separately.
+For remote images, the `src` property should be set to a string containing a URL, which can be [relative](#loaders) or [absolute](#domains). Because Next.js does not have access to remote files during the build proccess, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and optional [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually, as seen below.
 
-```js
+```jsx
 import Image from 'next/image'
-import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src={profilePic}
+        src="/me.png"
         alt="Picture of the author"
-        // width={500} automatically provided
-        // height={500} automatically provided
-        // blurDataURL="data:..." automatically provided
-        // placeholder="blur" // Optional blur-up while loading
+        width={500}
+        height={500}
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
+
+export default Home
 ```
 
-For remote images, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually.
+>(For additional information about the sizing requirement in next/image, [see here](#image-sizing))
 
-## Properties
+### Loaders
 
-[View all properties](/docs/api-reference/next/image.md) available to the `next/image` component.
+Note that in the example above, a relative URL (`"/me.png"`) is provided for a remote image. This is possible because of the `next/image` [loader](/docs/api-reference/next/image.md#loader) architecture. A loader is a function that takes the information you provide to the image component properties, and generates the URLs needed to serve optimized images. 
```

Comment:
```suggestion
In the example above, a relative URL (`"/me.png"`) is provided for a remote image. This is possible because of the `next/image` [loader](/docs/api-reference/next/image.md#loader) architecture. A loader is a function that takes the information you provide to the image component properties, and generates the URLs needed to serve optimized images. 
```

This sentence could be more clear, but I can't summarize it well... hmm.

"A loader is a function that takes the information you provide to the image component properties, and generates the URLs needed to serve optimized images."

Reply:
It is a little hard to summarize, isn't it? 😅  I came up with this:

***

"A loader is a function that generates the URLs for your image. It appends a root domain to your provided `src`, and generates multiple URLs to request the image at different sizes. These multiple URLs are used in the automatic [srcset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset) generation, so that visitors to your site will be served an image that is the right size for their viewport."

Reply:
Sounds great!